src/client/package.json

```diff
@@ -68,20 +67,18 @@
     "openfin-notifications": "1.20.0",
     "polished": "^4.1.1",
     "query-string": "^7.1.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-helmet": "^6.1.0",
```

Comment:
react-helmet was throwing a type error, and it's pretty poorly maintained.  i think we can do without it - we just directly modify the dom inside useEffect.  Since we're only updating head and not body, i don't think it will cause any problems.